src/components/flex/index.stories.tsx

```diff
@@ -1,26 +1,18 @@
 import React from 'react';
 import { Meta, StoryObj } from '@storybook/react';
 
-import Box from '../box';
+import { Flex } from './index';
 
-import Flex from './index';
+import { Box } from 'src';
 
 const meta: Meta<typeof Flex> = {
   title: 'Layout/Flex',
   component: Flex,
-
   args: {
     direction: 'row',
     align: 'center',
     justify: 'center',
-    children: [
-      <Box p="sm" bg="green.200" margin="sm" key="child-1">
-        Element 1
-      </Box>,
-      <Box p="md" bg="orange.300" margin="sm" key="child-2">
-        Element 2
-      </Box>,
-    ],
+    children: ['box1', 'box2'],
```

Comment:
You could also go for `boxes` and have the array defined directly in the mapping. Up to you.